assets/js/saaspricing-pricelist.js

```diff
@@ -0,0 +1,16 @@
+(function ($) {
+    "use strict";
+    $(window).on("elementor/frontend/init", function () {
+        elementorFrontend.hooks.addAction("frontend/element_ready/saaspricing-pricelist.default", function (scope, $) {
+            const priceListNumElements = $(scope).find('.saasp-pricelist-list-num');
+            let getTotalPriceElement = $(scope).find('.saasp-pricelist-total-num');
+            let PriceListAmount = 0;
+            priceListNumElements.each(function() {
+                PriceListAmount += parseInt($(this).text());
+            });
+            getTotalPriceElement.text(PriceListAmount);
+        })   
+    });
+})(jQuery);
+
```

Comment:
1. Use parseFloat instead of parseInt for better handling of decimal values.
2. Ensure to handle cases where the text might not be a number.
3. Use let instead of const for variables that are reassigned.